experiment/template/eVOLVER.py

```diff
@@ -482,6 +482,52 @@ def calc_growth_rate(self, vial, gr_start, elapsed_time):
         text_file.write("{0},{1}\n".format(elapsed_time, slope))
         text_file.close()
 
+    def tail_to_np(self, path, window=10):
+        f = open(path, 'rb')
```

Comment:
Open the file inside and not outside the function. I think it is the best option for our use case.